libs/server/src/lib/utils/utils.ts

```diff
@@ -131,7 +131,7 @@ export function fileExistsSync(filePath: string): boolean {
 }
 
 export function readAndParseJson(fullFilePath: string): any {
-  return JSON5.parse(readFileSync(fullFilePath).toString());
+  return JSON.parse(stripJsonComments(readFileSync(fullFilePath).toString()));
```

Comment:
mentioned this to Isaac and he wondered if it should also try catch and fallback to JSON5 if the regular parse fails

Reply:
This is the same "technique" used in Nx. If the workspace.json file includes anything related to json5 (trailing commas, etc) then Nx itself won't work anyway ☹️ 

I still want to work on this though, I think making this asynchronous will help with other things.